.changes/unreleased/Features-20240723-104221.yaml

```diff
@@ -1,5 +1,5 @@
 kind: Features
-body: render 'to' and 'to_columns' fields on foreign key constraints
+body: render 'to' and 'to_columns' fields on foreign key constraints, and bump dbt-common lower bound to 1.6
```

Comment:
```suggestion
bump dbt-common lower bound to 1.6
```

Reply:
I'm updating an existing changelog (since this change really should have been part of the initial implementation). Should I really drop the actual feature description from the changelog?

Reply:
I can also make a new changelog for this if that would be clearer

Reply:
ohhh my bad, I thought this was a new changelog